routes/users.js

```diff
@@ -826,16 +809,19 @@ router.delete_user = async function(user, action_owner_id, message){
         msg_destinations.push(user.email);
     }
 
-    let msg_del = CONFIG.message.deletion.join('\n').replace(/#UID#/g, user.uid).replace('#USER#', action_owner_id).replace('#MSG#', mail_message) + '\n' + CONFIG.message.footer.join('\n');
-    let msg_del_html = CONFIG.message.deletion_html.join('').replace(/#UID#/g, user.uid).replace('#USER#', action_owner_id).replace('#MSG#', mail_message) + '<br/>'+CONFIG.message.footer_html.join('<br/>');
-
-    let mailOptions = {
-        origin: MAIL_CONFIG.origin, // sender address
-        destinations:  msg_destinations, // list of receivers
-        subject: GENERAL_CONFIG.name + ' account deletion: ' + user.uid, // Subject line
-        message: msg_del, // plaintext body
-        html_message: msg_del_html // html body
-    };
+    try {
+        await utils.send_notif_mail({
+            'name': 'deletion',
```

Comment:
would be better to name template (and according name in config) user_deletion, deletion is a bit too generic

Reply:
https://github.com/genouest/genouestaccountmanager/pull/166/commits/b2733a6e9e4b6a6e5e7945620b5c048218ff134c